File: EmptyActivityApp/app/src/main/java/com/example/emptyactivityapp/MainActivity.java
package com.example.emptyactivityapp;

import android.os.Bundle;
import android.view.View;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.main_menu);
    }

    public void loadConstraintLayout(View view) {
        setContentView(R.layout.activity_main);
    }

    public void loadTableLayout(View view) {
        //setContentView(R.layout.my_table_layout);
    }
}

File: MySchool/app/src/main/java/com/example/myschool/MainActivity.kt
package com.example.myschool

import android.content.Intent
import android.os.Bundle
import android.os.Handler
import android.view.WindowManager
import android.view.animation.AnimationUtils
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    companion object {
        val SCREEN_TIMEOUT = 3000
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        window.setFlags(
            WindowManager.LayoutParams.FLAG_FULLSCREEN,
            WindowManager.LayoutParams.FLAG_FULLSCREEN
        )

        val topToBottomAnimation = AnimationUtils.loadAnimation(this, R.anim.from_top)
        val bottomToTopAnimation = AnimationUtils.loadAnimation(this, R.anim.from_bottom)

        bottomToTopAnimation.duration = 2800
        topToBottomAnimation.duration = 2800

        splashAppName.startAnimation(topToBottomAnimation)
        splashImage.startAnimation(topToBottomAnimation)
        splashLogoLine.startAnimation(bottomToTopAnimation)

        Handler().postDelayed(Runnable {
            val intent = Intent(this@MainActivity, LoginActivity::class.java)
            startActivity(intent)
            finish()
        }, SCREEN_TIMEOUT.toLong())
    }
}
